crates/holochain/src/conductor/handle.rs

```diff
@@ -944,8 +959,46 @@ impl ConductorHandleT for ConductorHandleImpl {
         Ok(installed_clone_cell)
     }
 
-    async fn destroy_clone_cell(self: Arc<Self>, _cell_id: CellId) -> ConductorResult<()> {
-        todo!()
+    async fn archive_clone_cell(
+        self: Arc<Self>,
+        payload: ArchiveCloneCellPayload,
+    ) -> ConductorResult<()> {
+        let ArchiveCloneCellPayload {
+            app_id,
+            clone_cell_id,
+        } = payload;
+        self.conductor
+            .archive_clone_cell(&app_id, &clone_cell_id)
+            .await?;
+        Ok(())
+    }
+
+    async fn restore_archived_clone_cell(
+        self: Arc<Self>,
+        payload: ArchiveCloneCellPayload,
+    ) -> ConductorResult<InstalledCell> {
+        let ArchiveCloneCellPayload {
+            app_id,
+            clone_cell_id,
+        } = payload;
+        let restored_cell = self
+            .conductor
+            .restore_clone_cell(&app_id, &clone_cell_id)
+            .await?;
+        self.create_and_add_initialized_cells_for_running_apps(self.clone(), Some(&app_id))
```

Comment:
Ah I see you added the line I expected above, here. Yeah, I see why you did this, since it's weird to pass a ConductorHandle down into the Conductor. I think ideally the ConductorHandle shouldn't do anything that the Conductor can't do itself. However, the whole ConductorHandle/Conductor thing has become a mess that needs to be reworked so I'm ok with this.

Reply:
I totally agree

Reply:
@abe-njama Please add this as a refactoring item for the post-beta phase.